tensorflow/docs_src/programmers_guide/saved_model.md

```diff
@@ -362,7 +362,7 @@ defined in:
 
 After training an `Estimator` model, you may want to create a service
 from that model that takes requests and returns a result.  You can run such a
-service locally on your machine or deploy it scalably in the cloud.
+service locally on your machine or deploy it scalable in the cloud.
```

Comment:
Probably better to just delete the word. Scalaby isn't a word, but scalable doesn't make sense here either.